actiontext/lib/action_text/attachment.rb

```diff
@@ -6,8 +6,9 @@ module ActionText
   class Attachment
     include Attachments::TrixConversion, Attachments::Minification, Attachments::Caching
 
-    TAG_NAME = "action-text-attachment"
-    SELECTOR = TAG_NAME
+    mattr_accessor :tag_name, default: "action-text-attachment"
+    mattr_accessor :selector, default: tag_name
```

Comment:
Is it worth keeping `selector` separate, or can we use `tag_name` everywhere we use `selector`?

Reply:
I would prefer to use tag_name everywhere. I don't see any use keeping them separate if they will never be different.